entrypoint.sh

```diff
@@ -9,5 +9,11 @@ if [ ! -z $namespace ]; then
 params="--namespace=$namespace"
 fi
 
+if [ ! -z "$OKTETO_CA_CERT" ]; then
+   echo "Custom certificate is provided"
+   echo "$OKTETO_CA_CERT" > /usr/local/share/ca-certificates/okteto_ca_cert
+   update-ca-certificates
+fi
```

Comment:
since we do this on every single action, do you think it might make sense to have a 'github action docker' image that already includes a script that does this?